pages/typeform-101.vue

```diff
@@ -0,0 +1,412 @@
+import FormBuilderComparisionTable from '../components/form-builders-comparision-table.vue'
+
+import FAQwithCategories from '../components/FAQwithCategories.vue'
+import WhyChooseFormester from '../components/jotform101/why-choose-formester.vue'
+import CallToActionSection from '../components/CallToActionSection.vue'
+
+import typeformFaqs from '@/faqs/typeform-101-faqs.js'
+import getSiteMeta from '@/utils/getSiteMeta'
+
+export default {
+  components: {
+    TypeformKeyFeatures,
+    StandOutFeatureSection,
+    PopularUseCase,
+    Limitations,
+    WhyChooseFormester,
+    FormBuilderComparisionTable,
+    FAQwithCategories,
+    CallToActionSection,
+  },
+  computed: {
+    meta() {
+      const metaData = {
+        type: 'website',
+        url: 'https://formester.com/typeform-101/',
+        title: 'Typeform 101 | Designing Forms with Visual Brilliance!',
+        description:
+          "Use Typeform to unleash your form-building powers. Formester provides a guide to exploring Typeform's capabilities, tips, and why it excels!",
+        mainImage: 'https://formester.com/typeform-101-hero-section.png', 
+        mainImageAlt: 'screenshot showing the hero section of typeform 101 page', 
+        keywords: [
+          'Typeform',
+          'Typeform101',
+          'Typeform Guide',
+          'Typeform101 Guide',
+          'Designing Forms with Visual Brilliance!',
+          'web forms for customer support',
+          'web forms productivity',
+          'form builder',
+          'formester',
+          'online forms',
+          'online web forms',
+          'formester web forms',
+          'form creator',
+          'form generator',
+          'online form',
+          'web form',
+          'online forms',
+          'web forms',
+          'create form',
+          'create forms',
+          'frequently asked questions',
+          'FAQ',
+        ],
+      }
+      return getSiteMeta(metaData)
+    },
+    faqsSchema() {
+      const allFaqs = []
+
+      this.typeformFaqs.forEach((c) => {
+        if (c.faqs) {
+          allFaqs.push(...c.faqs)
+        }
+      })
+
+      // Generate the FAQ schema
+      return allFaqs.map((faq) => {
+        return {
+          '@type': 'Question',
+          name: faq.question,
+          acceptedAnswer: {
+            '@type': 'Answer',
+            text: faq.answer,
+          },
+        }
+      })
+    },
+  },
+  head() {
+    return {
+      title: 'Typeform 101 | Designing Forms with Visual Brilliance!',
+      meta: [...this.meta],
+      link: [
+        {
+          hid: 'canonical',
+          rel: 'canonical',
+          href: 'https://formester.com/typeform-101/',
+        },
+      ],
+    }
+  },
+  jsonld() {
+    return {
+      '@context': 'http://schema.org',
+      '@graph': [
+        {
+          '@type': 'Corporation',
+          '@id': 'https://acornglobus.com',
+          name: 'Tyepform 101 | Guide to Form Building and Surveys with Jotform',
```

Comment:
'`Tyepform` 101 | Guide to Form Building and Surveys with `Jotform`' 

This should be `Typeform` there are two typos.
Also, check for any such case across the file using `ctrl + F` and searching for Jotform.

Reply:
done